src/eko/anomalous_dimensions/aem1.py

```diff
@@ -6,6 +6,7 @@
 
 from .. import constants
 from . import as1
+from . import asp1
```

Comment:
About this I would organize differently: a different subpackage rather than importing in the same. I would reconnect at the level of integration, in which we will switch between pol and unpol ad (either we compute the operator for one or for the other, the two are not connected).